CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+* Bug fix: avoid false positives on missing case-insensitive uniqueness indexes when using [Postgres `citext`](https://www.postgresql.org/docs/current/citext.html) strings (contributed by gee-forr)
```

Comment:
Please format that line, so that it doesn't exceed 80 characters.

Reply:
Done :)